Verbatim `util/div_mod.v` section of patches/prosa-v06-rocq93-official-proof-closure-common.patch
(sha256 6819bfa9ac0bb9cfd6bfc7182d57e22463cf7a57b38ce9af89c8e7de96868050), split out for classic official closures.
--- a/util/div_mod.v
+++ b/util/div_mod.v
@@ -1,5 +1,6 @@
 From mathcomp Require Export ssreflect ssrbool eqtype ssrnat seq fintype bigop div ssrfun.
 Require Export prosa.util.nat prosa.util.subadditivity.
+Set SsrOldRewriteGoalsOrder.
 
 (** * Additional Lemmas about [divn] and [modn] *)
 
